### ISSUE TITLE
Design solution for data migration issues

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#412]
Using repo exports to migrate data from v1 to v2 introduces a few challenges:
* Exports don't include users or orgs (and bulk import is missing a few features to support users/orgs/org members)
* Interim resource versions are not present in any exports and will be lost -- LOW PRIORITY FOR DATA MIGRATION
* There is no way to export HEAD -- HIGH PRIORITY
* Versioned collection references point to resource version UUIDs that are local, but these local UUIDs don't mean anything when you are moving to a new system -- importing sources and collections in the order of their created_on datetime would get us close, but is still not a complete solution

**Comment by aprilcurrier** [comment]
The issues with bulk imports have taken priority. To be continued after that is resolved.

**Comment by paynejd** [comment]
Drafting here:
https://docs.google.com/document/d/1Y2ahAGgZITA_f99fYYMuYFk_nmjWEyFb0Cbyt_pKxCE/edit#

**Comment by jamlung-ri** [comment]
Dependent on #420 - rolling over

**Comment by jamlung-ri** [comment]
This work can be started now that #420 is done

**Comment by snyaggarwal** [comment]
@rkorytkowski @paynejd I am very close to finishing the scripts to dump data from Mongo to Postgres. I wanted to highlight the process and ask opinions on few points. The way I am thinking of it is:

1. OCL basic data already imported on startup -- ocladmin, lookup-values (locales,maptypes, etc.) 
2. Run Bulk Imports for CIEL/PEPFAR/Others. _(this needs to run before any manual migration)_
_(below steps are manual migrations from v1 to v2)_
3. Export v1 data (in the order defined [here](https://docs.google.com/spreadsheets/d/1t8702D2xhLz9Ls2eOhxFkCzptxjGD1YVir4E_HRbB9Q/edit#gid=0))
4. Import exported data to v2 (in the order defined [here](https://docs.google.com/spreadsheets/d/1t8702D2xhLz9Ls2eOhxFkCzptxjGD1YVir4E_HRbB9Q/edit#gid=1377602598))

Questions:
1. Exports from mongo can be run on the latest data dump, and the file needs to be put on s3 for import to fetch. Is that ok? This will be manual?
2. How should we trigger import?
   - assuming via API, single API? Resource separated?

**Comment by paynejd** [comment]
* Add notice to the top of all `ocl_web` pages that points to the new service and that any edits will not be migrated

**Comment by jamlung-ri** [comment]
Next steps needed:
* Load CIEL dictionary on Staging/Prod
* Test migration run on Staging/Prod
* Re-migrate everything on Prod after test run
